addon/mixins/in-viewport.js

```diff
@@ -208,22 +212,28 @@ export default Mixin.create({
     });
   },
 
+  _triggerDidScrollDirectionHandler(event) {
+    const sensitivity = get(this, 'viewportScrollSensitivity') || 1;
+
+    this._debouncedEventHandler('_triggerDidScrollDirection', event.currentTarget, sensitivity);
```

Comment:
Curious what your thoughts are bw ever.currentTarget vs scrollableArea vs window in detecting scroll direction.  Perhaps irrelevant thus this might be a good fix?

Reply:
`scrollableArea` property may be null if it's not set in the component where the mixin is used. So I believe `currentTarget` is the best idea here.

Reply:
@latata If scrollableArea is not defined, then it defaults to `window`.  Did you happen to see in error wrt to this?